tests/unit/sparql/buildQuery.spec.ts

```diff
@@ -38,11 +42,34 @@ describe( 'buildQuery', () => {
 		const propertyId = 'P666';
 		const propertyValueRelation = PropertyValueRelation.Regardless;
 
-		expect( buildQuery( { condition: {
-			propertyId,
-			value: '',
-			propertyValueRelation,
-		} } ) ).toEqual( `SELECT ?item WHERE { ?item (p:${propertyId}/ps:${propertyId}) _:anyValue. }` );
+		expect( buildQuery( { conditions: [
+			{
+				propertyId,
+				value: '',
+				propertyValueRelation,
+			},
+		] } ) ).toEqual( `SELECT ?item WHERE { ?item (p:${propertyId}/ps:${propertyId}) _:anyValue. }` );
 	} );
 
+	it( 'builds a query from multiple conditions', () => {
```

Comment:
Yay! Can we have two more test-cases? I'm especially thinking about "matching" + "not matching" and "not matching" + "matching". As those "not matching" queries consist of two lines and I want to make sure that we get the syntax and meaning right.

Reply:
Done

Reply:
Mh, I think that those are added correctly, given the other code that we have. It is just that they are broken and not actually doing what they are supposed to do.

I'll create a new bug ticket for that. That is not blocking this task